.github/labeler.yml

```diff
@@ -1,3 +1,127 @@
+crate:core-path:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./logic/path/**
+crate:core-strategy:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./logic/strategy/**
+crate:hopr-internal-types:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./common/internal-types/**
+crate:hopr-primitive-types:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./common/primitive-types/**
+crate:hopli:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./hopli/**
+crate:hopr-db-entity:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./db/entity/**
+crate:hopr-db-api:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./db/api/**
+crate:hopr-db-migration:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./db/migration/**
+crate:hopr-crypto-random:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./crypto/random/**
+crate:hopr-crypto-sphinx:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./crypto/sphinx/**
+crate:hopr-crypto-types:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./crypto/types/**
+crate:hopr-lib:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./hopr/hopr-lib/**
+crate:hopr-metrics:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./misc/metrics/**
+crate:hopr-platform:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ./misc/platform/**
+crate:utils-db:
```

Comment:
Removed in our PR.

Reply:
The helper scripts syncs those properly, so just run it on your branch once this PR was merged.